app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :body, null: false
+  validates :user_id, null: false
+  validates :post_id, null: false
```

Comment:
You don't need to validate user_id and post_id for presence. The belongs_to association will enforce their presence. Same for user_id in post.rb